cli/client/client.go

```diff
@@ -113,6 +114,23 @@ func New(config config.Config) *RestClient {
 		return nil
 	})
 
+	// Verify the Sensu edition and update the sensuctl configuration if required
+	restyInst.OnAfterResponse(func(c *resty.Client, resp *resty.Response) error {
+		// Retrieve the Sensu edition from the response header
+		headerEdition := resp.Header().Get(types.EditionHeader)
+		if headerEdition == "" {
```

Comment:
Will there ever be a case that we don't return an edition header?

Reply:
I could see the case where someones run a recent sensuctl binary with an older sensu-backend.

Reply:
Ah good call!